ruby_on_rails-postgres/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: 'username'
+      POSTGRES_HOST_AUTH_METHOD: trust
+  web:
+    build: ./web
+    command: bash -c "rm -f tmp/pids/server.pid && yarn install --check-files && rake db:create db:migrate && bundle exec rails s -b 0.0.0.0"
```

Comment:
You could consider moving the `yarn install --check-files` into the `Dockerfile`. I'm not sure why you would need to run that install every time you bring the application up. This is really no different than the gems, which are handled in the `Dockerfile`

Reply:
Yes, you are sure, I will change